ingest/vendored/.gitrepo

```diff
@@ -6,7 +6,7 @@
 [subrepo]
 	remote = https://github.com/nextstrain/ingest
 	branch = main
-	commit = 9cfed8b1a93e881d85a71aef46e66730ca660523
-	parent = 143581871f1fdcc3ce0d020e1c3ddddffef2ffa9
+	commit = 258ab8ce898a88089bc88caee336f8d683a0e79a
+	parent = 3a2acd289683daf7450b6c9d9535df723438bc55
```

Comment:
Looks like this was copied over from yellow-fever instead of running the `git subrepo pull ingest/vendored` command? 

FWIW, if anyone tries to run the `git subrepo pull ingest/vendored`  in the future, it will error because this `parent` commit does not exist in this repo (see https://github.com/nextstrain/ingest/issues/21). To avoid this error, you can just replace this `parent` commit with the parent commit in this repo

```suggestion
	parent = 70659bd3887dadc44b8b086bced121fdf43d94d6
```

Reply:
> Looks like this was copied over from yellow-fever instead of running the `git subrepo pull ingest/vendored` command?

yeah, that command failed with some message about the parent commit not being in the repo, or something, and i was kinda flailing trying to get it to update. 

> FWIW, if anyone tries to run the `git subrepo pull ingest/vendored` in the future, it will error because this `parent` commit does not exist in this repo (see [nextstrain/ingest#21](https://github.com/nextstrain/ingest/issues/21)). To avoid this error, you can just replace this `parent` commit with the parent commit in this repo

👍

Reply:
Ah yeah, this failure mode is documented in the [ingest repo's README](https://github.com/nextstrain/ingest?tab=readme-ov-file#vendoring) but probably not clear enough. 

Running `git subrepo pull ingest/vendored --force` would have forced it to update